src/administration/from_scratch.md

```diff
@@ -18,12 +16,10 @@ The most recent stable version of PostgreSQL is 16 at the time of writing this g
 #### Install dependencies
 
 ```
-sudo apt install wget ca-certificates
-sudo apt install pkg-config libssl-dev libpq-dev postgresql
+sudo apt install -y wget ca-certificates pkg-config libssl-dev libpq-dev postgresql
 wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
 sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" >> /etc/apt/sources.list.d/pgdg.list'
 sudo apt update
-sudo apt install pkg-config libssl-dev libpq-dev postgresql
```

Comment:
Still looks wrong, you should keep this line at the bottom to install postgres etc after adding the apt repo, and also the apt install at the beginning for wget and ca-certificates (to be able to add the apt repo). Only the old line `sudo apt install pkg-config libssl-dev libpq-dev postgresql` was wrong and had to be removed.

Reply:
Oh :/